backend/.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
Any reason we're not using the same `.eslintrc.js` at the project root? Seems to be we should enforce the same rules project-wide.

Reply:
Different env: `browser: false, node: true`. Also, dependency on Jest is absent in the website `package.json`.